### DISCUSSION TITLE
破折号影响文本转语音功能

### DISCUSSION THREAD

**Discussion by SkyPan19** [ZuodaoTech/everyone-can-use-english#914] [Q&A]
母语使用破折号，文本转语音后，它就不显示破折号，而且会影响语音评估。

**Comment by an-lee** [comment]
没看明白，能详细描述一下在用哪个功能，每一步的输入是啥吗？最好能截图。

**Comment by SkyPan19** [comment]
语音转文本后，右边有音标处没有显示破折号，但这其实影响不大；但是在这句有破折号连接前后词汇的句子中，自我录音并用语音评估后，破折号前后的词汇评估不准确，即使我用手机AI播放该句子给Enjoy评估，依然破折号前后词汇评估不准确。
第二张图是手机AI朗读后的评估结果。
![屏幕截图 2024-07-29 203305](https://github.com/user-attachments/assets/f7468380-a88f-41bc-a8f2-632f49f86c82)
![屏幕截图 2024-07-29 203239](https://github.com/user-attachments/assets/2f029ae1-eb56-439e-baa7-59fd630dd830)

**Comment by an-lee** [comment]
不影响啊。评估结果是说你多读了一个 `is`

**Comment by an-lee** [comment]
如果你觉得影响，可以修改字幕，把破折号改成句号。
